layers/+completion/helm/funcs.el

```diff
@@ -7,12 +7,17 @@
 ;;
 ;; This file is not part of GNU Emacs.
 ;;
-;;; License: GPLv3
+;; License: GPLv3
 
-
+;;; Commentary:
 
-(defvar spacemacs--helm-popwin-mode nil
-  "Temp variable to store `popwin-mode''s value.")
+;;; Code:
+
+(require 'core-funcs)
+(require 'dash)
+(require 'dired)
```

Comment:
This is dangerous, in my case "dash" is not installed and subsequently Spacemacs stops booting.